src/bp/ui-studio/src/web/views/OneFlow/sidePanel/style.scss

```
@@ -1,5 +1,165 @@
 @import '../../../palette.scss';
 
+.sidePanel {
+  position: relative;
+  background-color: var(--white);
+  height: 100%;
+  overflow-y: auto;
+  font-size: 12px !important;
+
+  :global {
+    .bp3-tab {
+      font-size: 12px !important;
+    }
+
+    .bp3-editable-text-editing {
+      border-radius: 5px;
+      height: 20px;
+      margin: 5px 5px;
+    }
+
+    .bp3-editable-text.bp3-editable-text-editing::before {
+      box-shadow: unset;
+      border: solid 1px var(--ocean);
+    }
+
+    .bp3-editable-text input {
+      height: 100% !important;
+    }
+
+    .bp3-editable-text.bp3-editable-text-editing {
+      width: 100%;
+      min-width: 0px !important;
+    }
+
+    .bp3-editable-text:hover::before {
+      box-shadow: unset !important;
+    }
+
+    .bp3-tree-node.bp3-tree-node-selected > .bp3-tree-node-content {
+      color: var(--shark);
+      background-color: var(--hover-ocean);
+    }
+
+    .bp3-tree-node .bp3-tree-node {
+      margin: 0px var(--spacing-medium);
+
+      .bp3-tree-node-content {
+        border-radius: 5px;
+      }
+    }
+
+    .bp3-tree-root > .bp3-tree-node > .bp3-tree-node-content > .bp3-tree-node-icon {
+      display: none;
+    }
+
+    .bp3-tree-root .bp3-tree-node-list > :last-child {
+      :hover {
+        background-color: var(--hover-ocean);
+      }
+      .bp3-icon {
+        color: var(--ocean) !important;
+      }
+    }
+
+    .bp3-tree-node {
+      cursor: pointer;
+    }
+
+    .bp3-tree-root > .bp3-tree-node > .bp3-tree-node-content:hover {
+      background-color: var(--white);
+    }
+
+    .bp3-tree-root {
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+      margin-bottom: 0px !important;
+    }
+
+    .bp3-tree {
+      height: 100%;
+    }
+
+    .bp3-tree-node-caret-none {
+      min-width: 8px;
+    }
+
+    .bp3-tree-node-list,
+    .bp3-icon {
+      color: var(--shark);
+    }
+
+    .bp3-elevation-0 {
+      box-shadow: unset !important;
+    }
+
+    .bp3-editable-text-input {
+      width: 100% !important;
+    }
+  }
+}
+
+.addWorkflowNode {
+  color: var(--ocean);
+}
+
+.tree {
+  overflow: auto;
+  height: calc(100% - 98px);
+}
+
+.unfilteredTree {
+  :global {
+    .bp3-tree-root > :nth-last-child(2) {
+      pointer-events: none;
+      margin-top: auto;
+
+      .bp3-tree-node-content {
+        height: 18px;
+      }
+      hr {
+        color: var(--gray);
+      }
+    }
+  }
+}
+
+.topicsEmptyState {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 128px;
+  bottom: 128px;
+  margin: 0;
+}
+
+.topicsEmptyStateBlock {
+  z-index: 10;
+  text-align: center;
+  left: 0;
+  right: 0;
+  height: 160px;
+  margin: auto;
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+
+  :global {
+    .bp3-icon {
+      color: var(--ocean);
+      margin: auto;
+    }
+  }
+}
+
+.topicsEmptyStateText {
+  font-size: 12px;
+  color: #1a1e22;
+  margin: 0px 20px;
+  margin-top: var(--spacing-x-large);
```

instead of these two lines of margin
```suggestion
  margin: var(--spacing-x-large) var(--spacing-xx-large) 0;
```